### DISCUSSION TITLE
Can't connect onlyoffice with filerun in docker network

### DISCUSSION THREAD

**Discussion by hamx01** [ONLYOFFICE/Docker-DocumentServer#586] [Q&A]
Hi! I have docker network and onlyoffice container with filerun container in it.
This is OnlyOffice plugin configuration in Filerun:
![изображение](https://user-images.githubusercontent.com/101278038/218337011-9e855c6e-7aa2-40bc-b5c9-83fe813dc3ad.png)
And this is how I run container of onlyoffice:
![изображение](https://user-images.githubusercontent.com/101278038/218337195-a2861713-b46b-4e53-9b0f-d41760f35a5e.png)

The issue is that when I try to open some document with the onlyoffice I have that:
![изображение](https://user-images.githubusercontent.com/101278038/218337313-6bb3eb76-fdee-45ad-9742-16d60e6a7ba9.png)

But if I put the onlyoffice behind the reverse proxy like Nginx Proxy Manager it works great. I think this issue is related to headers.

I already tried to reach document server by IP, hostname, port forwarding and nothing worked, only reverse proxy.

I can send more screens and provide more info, if it only necessary

**Comment by SeanChengN** [comment]
Hi, I can't use filerun with onlyoffice, what is your Nginx Proxy Manager configuration? Below is mine:
![image](https://user-images.githubusercontent.com/20624068/226535895-da1d1216-55c3-4ccd-8b18-6e8b844aa025.png)
![image](https://user-images.githubusercontent.com/20624068/226535986-2de9a2a4-3639-4ac0-abaf-15515180370e.png)
![image](https://user-images.githubusercontent.com/20624068/226536055-819e2570-810b-4e25-98fb-bfab777f7880.png)
